velox/substrait/VeloxToSubstraitPlan.h

```diff
@@ -73,6 +73,31 @@ class VeloxToSubstraitPlanConvertor {
       const std::shared_ptr<const core::AggregationNode>& aggregateNode,
       ::substrait::AggregateRel* aggregateRel);
 
+  /// Convert Velox OrderBy Node into Substrait SortRel.
+  void toSubstrait(
+      google::protobuf::Arena& arena,
+      const std::shared_ptr<const core::OrderByNode>& orderByNode,
+      ::substrait::SortRel* sortRel);
+
+  /// Convert Velox TopN Node into Substrait SortRel->FetchRel.
+  void toSubstrait(
+      google::protobuf::Arena& arena,
+      const std::shared_ptr<const core::TopNNode>& topNNode,
+      ::substrait::FetchRel* fetchRel);
+
+  /// Helper function to process sortingKeys and sortingOrders.
```

Comment:
This comment is not informative. Would you update it to explain what kind of processing is done here?

Reply:
done.